src/App.js

```diff
@@ -16,7 +17,7 @@ function App() {
         <div className="flex-1">
           <Routes>
             <Route path="/login" element={<Login loggedIn={loggedIn} />} />
-            <Route path="/signup" element={<h1>Sign Up Page</h1>} />
+            <Route path="/signup" element={<Signup />} />
```

Comment:
Good job so far. I think you could also pass `loggedIn` to the `Signup` component. See my other comment to know why.